packages/joget/joget/upstream.yaml

```diff
@@ -0,0 +1,6 @@
+HelmRepo: https://jogetworkflow.github.io/helm-joget/charts
```

Comment:
While we're at it, it is best practice to add a newline to the end of these files (github is highlighting it as an issue for me).